strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java

```diff
@@ -387,7 +391,80 @@ private void getPackageInternal(String storageId,
             response.setStatus(INTERNAL_SERVER_ERROR.value());
         }
     }
+    
+    @ApiOperation(value = "Used to download the headers for a package")
+    @ApiResponses(value = { @ApiResponse(code = HttpURLConnection.HTTP_OK, message = "The package was downloaded successfully."),
+                            @ApiResponse(code = HttpURLConnection.HTTP_INTERNAL_ERROR, message = "An error occurred.") })
+    @PreAuthorize("hasAuthority('ARTIFACTS_RESOLVE')")
+    @RequestMapping(path = "{storageId}/{repositoryId}/{commandName:(?:download|package)}/{packageId}/{packageVersion}", method = RequestMethod.HEAD, produces = MediaType.APPLICATION_OCTET_STREAM)
```

Comment:
`produces = MediaType.APPLICATION_OCTET_STREAM`

That's not true :)

Reply:
@fuss86 this is really interesting thing, as you noticed HEAD requests should return same headers as GET requests does. So I maybe `content-type` header should be the same too?

Reply:
@sbespalov you're right. See https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.17 

@sanket407 please leave it as it is, as it follows standards. Thanks!